search/tests/test_indexing.py

```diff
@@ -36,3 +38,59 @@ def test_strip_xml(self):
         self.resource.save()
         self.assertTrue(self.count_results("you're it") == 1)
         self.assertTrue(self.count_results(xml) == 0)
+
+    def test_sorting(self):
```

Comment:
This test would be much easier to read (and shorter) if there were just as many resources created as sort options, with values that ensured each one would be first for a particular sort, then sort each way, asserting the order expected.

The test here takes more effort to understand than it has to.